lib/pbio/platform/debug/platform.c

```diff
@@ -248,16 +252,6 @@ void USART2_IRQHandler(void) {
     pbdrv_uart_stm32f4_ll_irq_handle_irq(UART_ID_0);
 }
 
-// HACK: we don't have a generic ioport interface yet so defining this function
-// in platform.c
-pbio_error_t pbdrv_ioport_get_iodev(pbio_port_t port, pbio_iodev_t **iodev) {
```

Comment:
Is this no longer used?

Reply:
ioport_lpf2.c defines it too, so there's duplicate compilation error if we kept this one. The note make it suggest it was going to be removed at that point.

Reply:
I think this is because the debug board was just using uartdev directly without the ioport. If we can, it would be nice to leave ioport disabled in the config and leave this here.

Reply:
The reason to enable `PBDRV_CONFIG_IOPORT` is so that `pbdrv_ioport_lpf2_process` is available from `uartdev`, but we could just wrap the whole service/poll part inside a `PBDRV_CONFIG_IOPORT` `#if` block instead.

Reply:
Another way to work around this would be to wrap `process_post(&pbdrv_ioport_lpf2_process, PROCESS_EVENT_xxx, data);` in a function in the `pbdrv_ioport_lpf2_` namespace and use the usual static inline patter if PBDRV_CONFIG_IOPORT is disabled.

Reply:
I've been trying to get rid of `processes.h` anyway since it leads to issues like this.

Reply:
Resolved by moving remaining key point to the issue.